index.html

```diff
@@ -273,31 +273,23 @@ <h3 class="heading-tertiary">Contact me</h3>
           with it
         </p>
       </div>
-      <form id="contact-form" method="POST" class="contact-form" action="https://formspree.io/f/xzbyaeyk">
+      <form action="https://formspree.io/f/mzbyjjjd" class="contact_form" method="post">
         <ul>
           <li>
-            <input id="user-name" type="text" pattern="[A-Za-z]+" class="contact-element" name="user-name" placeholder="Abenezer-Tilahun" maxlength="30" required/>
-            <small></small>
+            <label for="name">Name:</label>
+            <input class = "contact_name" type="text" id="name" name="name" placeholder="Enter your name here" maxlength="30" required />
           </li>
           <li>
-            <input id="email" type="email" class="contact-element" name="user-email" placeholder="Abenezermesafint2128@gmail.com" required/>
+            <label for="email">Email:</label>
+            <input class="email_address" type="email" id="email" name="email" placeholder="Enter your email here" required />
           </li>
-
           <li>
-            <textarea
-              id="user-message"
-              name="user-message"
-              class="contact-element"
-              cols="30"
-              rows="10"
-              placeholder="Write your message here"
-              maxlength="500"
-              required>
-            </textarea>
+            <label for="message">Message:</label>
+            <textarea class="contact_message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
           </li>
-          <li>
-            <p class="show-msg">Please make sure the email is all lower case!</p>
-            <input id="submit" type="submit" value="Get in touch" class="button btn-contact"/>
+          <li class="form_li_submit">
+            <p class="validation_message"></p>
+            <input type="submit" class="submit_btn" value="Get In Touch" />
           </li>
```

Comment:
## BEAUTIFUL DESIGN 👏

- [ ] You did a great job so far. However, there is just one issue in your design and I am sure you can fix it 😉. Please kindly fix the width of the validation message which is changing the width of the form. You can see the screenshot below 📸, it's getting wider

**Not submitted** 
![image](https://user-images.githubusercontent.com/69011963/152169367-2feb7dee-eba0-4d46-920e-1689cc125112.png)

**After I click the button**
![image](https://user-images.githubusercontent.com/69011963/152169468-05d24ada-72e3-45c3-b7bd-c7583b822ce4.png)

Reply:
Done thanks for suggestion.